CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-## Version 0.7.2
+Version 0.7.2
+==============
```

Comment:
Can you change the title formatting back to what it was? The old formatting was correct for markdown.